config/aso/credentials.yaml

```diff
@@ -3,7 +3,10 @@ kind: Secret
 metadata:
   name: aso-controller-settings
 type: Opaque
-data:
+stringData:
+  AZURE_AUTHORITY_HOST: ${AZURE_AUTHORITY_HOST:=""}
```

Comment:
Where should we document these? I don't see any existing docs about how to define this kind of config today.

Reply:
I've been using CAPZ to target _management.usgovcloudapi.net_  using a local  infrastructure-components.yaml with those same 3 secret values set.

I think you already tested what needed to be tested for this PR if `clusterctl init` resulted in a pacthed aso secret ✔️.

I will be able to delete some code on my side if this goes in, thanks 😄

Reply:
Not sure where in this repo, but this might be a good place to also mention them https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/book/src/user/quick-start.md?plain=1#L455

Reply:
@dtzar Did you know of a good spot to document these settings? Or could we defer this for now here and let that be part of a larger docs refactor?